letter_opener_web.gemspec

```diff
@@ -23,8 +23,8 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'letter_opener', '~> 1.0'
   gem.add_dependency 'railties', '>= 3.2'
 
```

Comment:
If you know it no longer works with older rubies, you might want to set a minimum ruby version in the gemspec:

```
  gem.required_ruby_version     = ">= x.x.x"
```